skimage/io/collection.py

```diff
@@ -100,19 +100,18 @@ class ImageCollection:
 
     Parameters
     ----------
-    load_pattern : str or list of str
-        Pattern string or list of strings to load. The filename path can be
-        absolute or relative.
+    load_pattern : str or sequence
+        A glob-like pattern or sequence of patterns to load files. The pattern can be
+        absolute or relative. If a sequence of objects other than strings is passed,
+        e.g. a range of numbers, each item will be passed directly to `load_func`.
```

Comment:
each item will be passed independently, each item will be passed sequentially, ... some way to show that they are passed one by one. Could also rewrite as: `load_func` will be applied to each element of the load_pattern. That's quite clear.